test/packages/EABase/include/Common/EABase/config/eacompiler.h

```diff
@@ -726,9 +726,9 @@
 			// Extern template is supported.
 		#elif defined(EA_COMPILER_CPP11_ENABLED) && defined(__EDG_VERSION__) && (__EDG_VERSION__ >= 401)  // EDG 4.1+.
 			// supported.
-		#elif defined(EA_COMPILER_CPP11_ENABLED) && defined(__clang__) && defined(__APPLE__) && (EA_COMPILER_VERSION >= 401)
+		#elif defined(EA_COMPILER_CPP11_ENABLED) && defined(__clang__) && defined(__apple_build_version__) && (EA_COMPILER_VERSION >= 401)
```

Comment:
Ahh I remember the problem now... if I have clang3.9 running on mac OS X, this is true:
defined(__clang__) && defined(__APPLE__)
because it is the case that the compiler is clang and the platform is apple.